code/datums/status_effects/debuffs.dm

```diff
@@ -740,3 +740,46 @@
 	name = "TO THE STARS AND BEYOND!"
 	desc = "I must go, my people need me!"
 	icon_state = "high"
+
+/datum/status_effect/fake_virus
+	id = "fake_virus"
+	duration = 1800//3 minutes
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 1
+	alert_type = null
+	var/msg_stage = 0//so you dont get the most intense messages immediately
+
+/datum/status_effect/fake_virus/tick()
```

Comment:
wouldnt it be obvious from the amount of symptoms that this isnt a real virus?

Reply:
That may be the case and I was going to address it but it's possible that pretty much this whole PR will have to be rewritten anyways should the bounty author want it done differently

Reply:
After testing a bit more I've decided that it's actually close enough to resemble a real virus that this won't be a big issue